examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
ah I see what you mean - let me look at it and see how that works

happy new year :)

On Thu, Dec 31, 2015 at 4:30 PM, Mainak Jas <notifications@github.com>
wrote:

> In examples/time_frequency/plot_compute_raw_data_spectrum.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48672856>:
>
> > @@ -70,3 +71,18 @@
> >
> >  ax.set_title('Four left-temporal magnetometers')
> >  plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
> > +
> > +# Alternatively, you may also create PSDs from Raw objects with psd_XXX
> > +f, ax = plt.subplots()
> > +psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
>
> hmm ... did you look at make_projector_info? Isn't that doing a picks on
> the projector? Also, you do have access to the channels in the projector
> object. See: raw.info['projs'][0]['data']['col_names'], which can be used
> to throw an error. It's late in India, will look at the thread tomorrow :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48672856>.
>